spark/src/main/resources/R/zeppelin_sparkr.R

```diff
@@ -31,7 +31,7 @@ print(paste("LibPath ", libPath))
 library(SparkR)
 
 
-SparkR:::connectBackend("localhost", port, 6000)
+SparkR:::connectBackend("localhost", port)
```

Comment:
I mean this code change is due to SparkR api change in spark 2.1.  And I suspect whether sparkr shell has the same issue, if yes then I don't think zeppelin need to fix this.